src/FakeItEasy/Analysis/MustUseReturnValueAttribute.cs

```diff
@@ -0,0 +1,25 @@
+﻿namespace FakeItEasy.Analysis
+{
+    using System;
+
+    /// <summary>
+    /// Indicates that the return value of a method should not be ignored.

```

Comment:
Perhaps this should read

> Indicates that the return value of a method must be used.

in order to tie-in with the name of the attribute?